python-checks/src/test/resources/checks/verifiedSslTlsCertificateCheckFNs.py

```diff
@@ -0,0 +1,51 @@
+def fixupRequestBulletPoint3WallOfFalseNegatives():
```

Comment:
do we really need all this test cases to show the FN? Moreover I wouldn't create a separate file for it